MapboxCoreNavigation/RouteController.swift

```diff
@@ -330,6 +339,50 @@ extension RouteController: CLLocationManagerDelegate {
         }
     }
     
+    func checkForFasterRoute(from location: CLLocation) {
+        guard let currentUpcomingManeuver = routeProgress.currentLegProgress.upComingStep else { return }
+        
+        guard let lastLocationDate = lastLocationDate else {
+            self.lastLocationDate = location.timestamp
+            return
+        }
+
+        // Only check ever 2 minutes for faster route
+        guard location.timestamp.timeIntervalSince(lastLocationDate) >= 120 else { return }
```

Comment:
Let's change this to once every 5 minutes. Eventually, we could adapt this for longer segments (e.g. >50 mile highway driving) and space this out to once every 10-15 minutes.